carbonmark-api/src/app.constants.ts

```diff
@@ -56,6 +61,54 @@ export const TOKEN_ADDRESSES = {
   },
 };
 
-export const RPC_URLS = {
+export const URLS = {
```

Comment:
but what **kind** of URLs

Reply:
changed back, not sure why I changed this to begin with